koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -826,7 +843,7 @@ def generate_monthly_cost_json_object(self, distribution, distributed_cost):
         )
 
     def populate_monthly_cost(
-        self, cost_type, rate_type, rate, start_date, end_date, cluster_id, cluster_alias, distribution
+        self, cost_type, rate_type, rate, start_date, end_date, cluster_id, cluster_alias, distribution, provider_uuid
```

Comment:
The `provider_uuid` was added to numerous places in this file to ensure that any newly created `OCPUsageLineItemDailySummary` entries are associated with the proper source.